min.js

```diff
@@ -0,0 +1 @@
+const minimum = (a, b) => (a < b ? a : b);
```

Comment:
I've seen this already (a different version), why don't add this to the previous PR.